models/base_model.py

```diff
@@ -440,6 +444,12 @@ def get_current_D_accuracies(self):
                 accuracies[name] = float(getattr(self, name))  # float(...) works for both scalar tensor and float number
         return accuracies
 
+    def get_current_APA_prob(self):
+        current_APA_prob=OrderedDict()
+        current_APA_prob['D_loss'] = float(self.D_loss.adaptive_pseudo_augmentation_p)
```

Comment:
`D_loss` gets reported in the graph legend, though I believe we are visualizing the probability, right ?